msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -9,8 +9,9 @@ ms.date: 05/20/2019
 # Developer preview manifest schema for Microsoft Teams
 
 > [!NOTE]
-> For more information on the program and how you can join,see [Developer Preview](~/resources/dev-preview/developer-preview-intro.md).
-> If you are not using the developer preview you should not be using this version of the manifest. See [Reference: Manifest schema for Microsoft Teams](~/resources/schema/manifest-schema.md) for the public version of the manifest.
+> * For more information on the program and how you can join,see [Developer Preview](~/resources/dev-preview/developer-preview-intro.md).
+> * If you are not using the developer preview you should not be using this version of the manifest. See [Reference: Manifest schema for Microsoft Teams](~/resources/schema/manifest-schema.md) for the public version of the manifest.
+> * App customization is not supported for sideloaded and LOB apps. You cannot test the feature by sideloading the app.
```

Comment:
Suggest moving this under the `configurableProperties` section where it would be more in context with the feature. Also, see my previous comment clarifying if this feature is just for apps published to the Teams store.

Reply:
Updated suggsetion.